CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Misc Improvements
 
+  * [#5103](https://github.com/osmosis-labs/osmosis/pull/5103) Run `make format`
```

Comment:
Let's only put changes in the changelog that clients would like to know about. e.g. API breaks, bug fixes etc

```suggestion
```

Reply:
This doesn't block merge. Will update the changelog pre-release